packages/events/src/EventBoundary.ts

```diff
@@ -6,6 +6,7 @@ import { Point } from '@pixi/math';
 import type { Cursor, FederatedEventTarget } from './FederatedEventTarget';
 import type { DisplayObject } from '@pixi/display';
 import type { FederatedEvent } from './FederatedEvent';
+import { EventEmitter } from '@pixi/utils/src';
```

Comment:
```suggestion
import { EventEmitter } from '@pixi/utils';
```

Reply:
I realize that sometimes VSCode auto-completes this. Be mindful not to do it tho.

Reply:
Fixed 👍